docs/products/opensearch/troubleshooting/troubleshooting-opensearch-dashboards.rst

```diff
@@ -0,0 +1,14 @@
+Troubleshoot OpenSearch® Dashboards
+==================================
+
+OpenSearch® Dashboard is unavailable
+---------------------------------------
+
+OpenSearch Dashboards requires that the version of OpenSearch on your cluster be the same. If OpenSearch Dashboards is unavailable, you will see the following error message in the OpenSearch Dashboards logs:
+
+.. code:: text
```

Comment:
is this what you mean by using a code style. When i try to preview it on github. it does not look like it is getting formatted. 

Also how do ignore these vale errors? as this is what the log message looks like.

Reply:
@RommelLayco 
I included it in a code-block. Your approach is also accurate; when formatting after "..code::", ensure to create a line break and indent the text.
I noticed that the Table of Contents (TOC) entry was missing for this topic. I've included the TOC entry and updated the code formatting accordingly.